casdoorsdk/base.go

```diff
@@ -175,8 +175,12 @@ func DoPostBytesRaw(url string, contentType string, body io.Reader) ([]byte, err
 // modifyUser is an encapsulation of user CUD(Create, Update, Delete) operations.
 // possible actions are `add-user`, `update-user`, `delete-user`,
 func modifyUser(action string, user *User, columns []string) (*Response, bool, error) {
+	return modifyUserByID(action, user.GetID(), user, columns)
```

Comment:
All `ID` should be changed to `Id`

Reply:
Should we follow https://google.github.io/styleguide/go/decisions#initialisms?

Reply:
No